### PR TITLE
Add Dock specific config

### PR DESCRIPTION
**Pull request by lovesh**
Signed-off-by: lovesh <lovesh.bond@gmail.com>

### CODE REVIEW

File: packages/apps-config/src/api/spec/dock.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'AccountId',
+  LookupSource: 'AccountId',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/dock.ts:9]: There is no `RefCount` override, so I'm assuming this is all based off the released Substrate 2.0 (final)?

Comment (lovesh) [reply packages/apps-config/src/api/spec/dock.ts:9]: Yes.

**Review by jacogr — APPROVED** [review]

**Comment by jacogr** [comment]
Thank you!

**Comment by lovesh** [comment]
@jacogr Thank you for merging!

**Pull request merged**